src/ru/nsu/ccfit/zuev/osu/game/cursor/main/CursorEntity.java

```diff
@@ -27,10 +28,10 @@ public CursorEntity() {
             particleOffsetX = -trailTex.getWidth() / 2f;
             particleOffsetY = -trailTex.getHeight() / 2f;
 
+            var spawnRate = (int) GlobalManager.getInstance().getMainActivity().getRefreshRate() * 2;
```

Comment:
Is there a reason why this uses double the display refresh rate?

Reply:
Actually this should be clamp according to the acceleration but that'll require more modifications than that, to handle the non-interpolation between frames I use refresh rate 2x.